### DISCUSSION TITLE
What are you using Jan for?

### DISCUSSION THREAD

**Discussion by Ssstars** [janhq/jan#1365] [General]
Hey folks
Massive last few days and the feedback has been amazing. 
Given we don't track you 👀 we're asking. direct.

**Comment by hwc0x01** [comment]
Waiting for: RAG! audio/video transcription. prompt engineering tools. mobile app.
Current use: learning a bunch about LLM stuff, bit of text gen, bit of prompt engineering practice, and messing around.

**Reply by TomLucidor** [reply]
Seconded, waiting for integration with LogSeq and other smart note takers... and also possible web crawling (e.g. blog and documentation) integration.

**Comment by Brog33** [comment]
uncensored models when?

**Reply by techogre** [reply]
You can download the model outside the app. Here are instructions:  https://jan.ai/guides/using-models/import-manually/

**Comment by PiotrWarzachowski** [comment]
I wanted to contribute a bit by making this open source project, where **I used a ChatGPT to summarize PDF documents**, you can **contribute to change it to jan** if you want, here is my repo : https://github.com/PiotrWarzachowski/DocMind Feel free to check it out, **its public on vercel**. You can leave a star ⭐ if you want 💓. Have a good year

**Reply by TomLucidor** [reply]
How is this different from GPT4All, Khoj, and PrivateGPT? Also why can't most of them handle books and long docs?

**Comment by Techeters** [comment]
Jan is great for privacy.  It's so hard to keep information away from big tech.  Jan makes it simple and easy without having deep technical knowledge.

**Reply by TomLucidor** [reply]
Easier to install than PrivateGPT too

**Comment by FatMixG** [comment]
Instructions for using llava 1.5-7b-q5 locally (if that's even possible) or instructions installing & using.

**Comment by TomLucidor** [comment]
Right now Ollama came out as well which is kinda surprising how fast things are moving, but there are always room between model servers and RAG + Roleplay + Web Agents
